tests/asserts/final/publication/geoserver_util.py

```diff
@@ -1,6 +1,9 @@
-def is_complete_in_workspace_wms_instance(wms_instance, name):
+def is_complete_in_workspace_wms_instance(wms_instance, name, validate_metadata_url=False):
```

Comment:
I prefer to have `validate_metadata_url` as required keyword argument without default value